gdc_client/download/parser.py

```diff
@@ -123,10 +127,13 @@ def download(parser, args):
         log.debug('Downloading big files...')
 
         # create URLs to send to parcel for download
-        bigs = [ urlparse.urljoin(client.data_uri, b) for b in bigs ]
-        downloaded_files, big_error_dict = client.download_files(bigs)
+        params = ('latest',) if args.latest else ()
+        bigs = [
+            urlparse.urljoin(client.data_uri, url_with_params(b, *params))
```

Comment:
Be careful with urljoin. It does some URL validation. I think this should be OK if we are re-formatting urls as we get them through the `--server` flag.
```python
In [1]: from urlparse import urljoin

In [2]: urljoin('a.b.c', 'data')
Out[2]: 'data'

In [3]: urljoin('https://a.b.c', 'data')
Out[3]: 'https://a.b.c/data'
```

Reply:
I think we do `fix_url` stuff, which _should_ handle this.